packages/application-components/src/components/page-content-containers/page-content-wide/page-content-wide.tsx

```diff
@@ -55,10 +55,33 @@ const Container = styled.div`
 
 function PageContentWide(props: TPageContentWide) {
   const [leftChild, rightChild] = Children.toArray(props.children);
+  const countChildren = Children.count(props.children);
+
+  // if there's 1 column and more than 1 child
+  const isColumnsOneAndMoreThanOneChild =
+    props.columns === '1' && countChildren > 1;
+
+  useWarning(
+    !isColumnsOneAndMoreThanOneChild,
+    'PageContentWide: This component has more than 1 child. Only the first child will be rendered.'
+  );
+
+  // if there's 1/1 or 2/1 columns and only 1 child
+  const isColumnsTwoAndOneChild =
+    (props.columns === '1/1' || props.columns === '2/1') && countChildren === 1;
```

Comment:
Wasn't sure if we want to show a warning in this case, but it made sense to me that if only one child is passed it would be usually by a mistake. But maybe I'm wrong?

Reply:
Technically, the second column is not required when using two columns.
I've found some use cases in MC frontend repository where we're providing just one column in a two columns layout.
I'm not very sure if that was intentional, but I think it's safer for us to now apply that restriction.

I believe it would be better to keep the previous one and just add another for the use case of receiving more that two children on a two column layout:
```js
  const [leftChild, rightChild] = Children.toArray(props.children);
  const childrenCount = Children.count(props.children);

  useWarning(
    props.columns === '1' && childrenCount > 1,
    'PageContentWide: This component only renders its first children when using a single column but you provided more that one.'
  );

  useWarning(
    props.columns !== '1' && childrenCount > 2,
    'PageContentWide: This component only renders its first two children when using a two columns layout but you provided more that two.'
  );
```

Reply:
Thanks for the feedback! 💡  Here's the change - [aaa1609](https://github.com/commercetools/merchant-center-application-kit/pull/3127/commits/aaa16098ceb3af35f12d773f93f2a4e22910a6ce).